templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    <li><a href="{% url "taxi:index" %}">Home page</a></li>
```

Comment:
make sure that you maintain the same style (single or double quotes) consistently `"taxi:index"`